pkgs/by-name/go/goto/package.nix

```diff
@@ -0,0 +1,65 @@
+{
+  coreutils,
+  fetchFromGitHub,
+  gawk,
+  goto,
+  lib,
+  nix-update-script,
+  runCommand,
+  stdenvNoCC,
+}:
+
+let
+  owner = "iridakos";
+  rev = "b7fda54e0817b9cb47e22a78bd00b4571011cf58";
```

Comment:
Move this to `fetchFromGithub`, and use the following:
```
owner = "iridakos";
tag = "v${finalAttrs.version}";
```

Reply:
The problem is that the 2.1.0 release is not tagged, so I have to resort to a commit hash.

Reply:
Just add `# no tags` comment, but you can move the 2 two within `fetchFromGithub`